### ISSUE TITLE
Add i18n support

### ISSUE THREAD

**Issue by jomaxx** [jordalgo/insult-jason#4]

**Comment by abe4mvp** [comment]
👍  i'd like to make my insults ```en español```

**Comment by palmerj3** [comment]
Jason in Japanese is ジェイソン

**Comment by scottsheffield** [comment]
I think we need to have a proper investigation into just how many languages Jason can understand before we start scoping for full i18n support.

**Comment by ccarpita** [comment]
I disagree with the assumption that Jason needs to understand the insults hurled in his direction by people all over the world.
